pkg/controllers/tagging/tagging_controller.go

```diff
@@ -116,30 +116,32 @@ func NewTaggingController(
 		return nil, err
 	}
 
-	var rateLimiter workqueue.RateLimiter
+	var rateLimiter workqueue.TypedRateLimiter[any]
```

Comment:
What's the benefit of this implementation if we're using `any` as the type variable?